app/app.go

```diff
@@ -255,7 +256,12 @@ func NewAxelarApp(
 	SetKeeper(keepers, initAxelarIBCKeeper(keepers))
 
 	if IsWasmEnabled() {
-		SetKeeper(keepers, initWasmKeeper(encodingConfig, keys, keepers, bApp, appOpts, wasmOpts, homePath))
+		if wasmDir == "" {
+			dbDir := cast.ToString(appOpts.Get("db_dir"))
```

Comment:
why not set wasm dir inside `initWasmKeeper`? the function already takes in all the necessary params

Reply:
That is what I did initially, but then Chris asked me to move it outside :)

Reply:
nvm, we would need an additional param to initWasmKeeper, make sense to move outside